src/pages/hoicyoo/Components/Nav.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+
+import "./Nav.scss";
+
+class Nav extends Component {
+  render() {
+    return (
+      <div className="title_area_wrapper">
```

Comment:
sass 선택자 분리를 위해서 컴포넌트의 최 상단 className은 class 명과 같게 해주셔야 합니다. 
sass로 nesting해서 css 선택자를 분리하는 이유를 생각해보아요!
```suggestion
      <div className="nav">
```